frontend/src/features/idir/search/components/IDIRPermitSearchRowActions.tsx

```diff
@@ -131,11 +134,11 @@ export const IDIRPermitSearchRowActions = ({
     if (selectedOption === PERMIT_ACTION_TYPES.RESEND) {
       setOpenResendDialog(() => true);
     } else if (selectedOption === PERMIT_ACTION_TYPES.VIEW_RECEIPT) {
-      viewReceiptPdf(permitId);
+      viewReceiptPdf(permitId, companyId);
     } else if (selectedOption === PERMIT_ACTION_TYPES.VOID_REVOKE) {
-      navigate(`${routes.PERMITS_ROUTES.VOID(permitId)}`);
+      navigate(`${routes.PERMITS_ROUTES.VOID(companyId, permitId)}`);
```

Comment:
Similarly here, what would be the behaviour if companyId passed on was undefined?